_posts/en/newsletters/2020-01-15-newsletter.md

```diff
@@ -0,0 +1,139 @@
+    the activation period passes without activation (but also without
+    any blocking problems being discovered), new releases can enable
+    this option by default for the remaining 24 months of the activation
+    period.
+
+    In the responses to the post, [Jorge Timón][timon sf] and [Luke
+    Dashjr][dashjr sf] both proposed that any BIP8-like mechanism use
+    mandatory versionbits signaling leading up to the flag day (similar
+    to how [BIP148][] proposed to activate segwit); Corallo
+    [notes][corallo reply timon] that this conflicts with the third and
+    fourth goals.  Jeremy Rubin provides a [quick analysis][rubin sf] of
+    his previous [spork proposal][spork vid] (see [Newsletter #32][spork
+    summary]) in the context of the five goals.  Anthony Towns provides
+    [lucid commentary][towns sf] on several aspects of Corallo's
+    proposal and Timón's response.
+
+    No clear conclusion was reached in the thread and we suspect
+    discussion about activation methods will continue.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #17578][] updates the `getaddressinfo` RPC to return an
+  array of strings corresponding to the labels used by an address.
+  Previously, it returned an array of objects where each label contained
+  fields for its name and its purpose, where purpose corresponded to the
+  part of the wallet that generated the address.  Users who need the old
+  behavior can use the `-deprecatedrpc=labelspurpose` configuration
+  parameter until Bitcoin Core 0.21.
+
+- [Bitcoin Core #16373][] causes the `bumpfee` RPC used for
+  [Replace-by-Fee][topic rbf] fee bumping to return a [Partially Signed
+  Bitcoin Transaction][topic psbt] (PSBT) when the user attempts to fee
+  bump a transaction using a wallet with disabled private keys.  The
+  PSBT can then be copied to an external wallet (such as a hardware
+  device or cold wallet) for signing.
+
+- [Bitcoin Core #17621][] fixes a potential privacy leak for users of
+  the `avoid_reuse` wallet flag.  This flag prevents the wallet from
+  spending bitcoins received to an address the wallet has already used
+  for spending (see [Newsletter #52][]).  This enhances privacy by
+  preventing otherwise-unrelated transactions from being associated
+  together because they all involve the same address.  However, Bitcoin
+  Core currently monitors for payments to any of its public keys using
+  any of several different address formats.  This meant that multiple
+  payments to the same public key---but with different addresses (e.g.
+  P2PKH, P2WKH, P2SH-P2WPKH)---could be associated with each other on
+  the block chain even though the `avoid_reuse` behavior thought it
```

Comment:
"the avoid_reuse behavior thought..." seems a bit imprecise to me. It's the user who thinks he/she isn't reusing addresses. The "avoid_reuse behaviour" can't have thoughts!

Reply:
fixed